notify/src/component/filter-container/Filter.jsx

```diff
@@ -0,0 +1,107 @@
+import { useFilterContext } from "../../context/filter-context";
+import { useNotesContext } from "../../context/notes-context";
+import "./Filter.css";
+const Filter = () => {
+  const { filterState, filterDispatch } = useFilterContext();
+  const { notesState } = useNotesContext();
+  return (
+    <div className="filter-container">
+      <div className="filter-options">
+        <h4> Sort</h4>
+        <label>
+          <input
+            type="radio"
+            name="radio"
+            checked={filterState.sort === "OLD"}
+            onChange={() => filterDispatch({ type: "OLD_TO_NEW" })}
+          />
+          OLD
+        </label>
+        <label>
+          <input
+            type="radio"
+            name="radio"
+            checked={filterState.sort === "NEW"}
+            onChange={() => filterDispatch({ type: "NEW_TO_OLD" })}
+          />
+          NEW
+        </label>
+      </div>
+      <div className="filter-options">
+        <h4> Priority</h4>
+        <label>
+          <input
```

Comment:
This `label` tag is useless unless you provide an `htmlFor` attribute that matches the `id` attribute of the `input`